addons/godot-xr-tools/assets/PlayerBody.gd

```diff
@@ -44,6 +44,12 @@ export (NodePath) var origin = null
 ## Path to the ARVRCamera node
 export (NodePath) var camera = null
 
+# Set our collision layer
+export (int, LAYERS_3D_PHYSICS) var collision_layer = 1 << 19 setget set_collision_layer, get_collision_layer
```

Comment:
Seeing we've got this in a bunch of places now, I think we should make the default values a constant in our helper script. Something like:
```
const XR_TOOLS_PLAYER_LAYER = 1 << 19
const XR_TOOLS_DEFAULT_MASK = 1023
```
etc.

Reply:
The only place in code for these constants is now PlayerBody.gd (with this pull request). We do however have collision values encoded into scene nodes:
- PlayerBody/KinematicBody layer/mask (1<<19 and 1023 - now controlled by PlayerBody.gd)
- Function_Pickup layer/mask (1<<19 and 1023)
- Function_Pointer mask (1<<20 - 1)
- Function_Teleport layer/mask (1<<19 and 1023)
- Function_Wind_Movement mask (1<<20)

Reply:
That's indeed the thing with declarative development through nodes, you can't really add in constants that way :)